.github/workflows/publish.yaml

```diff
@@ -4,7 +4,8 @@ name: Publish
 
 on:
   pull_request:
-    branches: [ main ]
+    branches:
+      - '**'
```

Comment:
I can only guess how/when this is used. I'm not sure if I should have just left this alone.

Reply:
I would leave the publish workflow as is (there are other checks - within the publishing tool - that also check for `main`).